spec/models/topic_category_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe TopicCategory do
+  subject { FactoryGirl.create(:topic_category) }
+
+  describe "#best_match" do
+    it "selects the best match from a list of options" do
+      options = ["Spike", "Vampire3", "Drusilla", "Vampire2", "Harmony"]
+      expect(subject.best_match options).to eq "Vampire2"
+    end
+
+    it "ignores case, whitespace, and punctuation when matching" do
+      options = ["Vampire3", "VamPire 2!"]
+      expect(subject.best_match options).to eq "VamPire 2!"
+    end
+
+    after do
+      FactoryGirl.reload
```

Comment:
This is resetting the sequence numbers  after each test - otherwise I get Vampires 5-8 in whatever test runs second.

Thoughtbot recommends naming models explicitly rather than relying on sequence numbers when they're going to be referenced in test assertions, but it considering the tree of the topic-related models that needs to get created here it seemed worthwhile to rely on the factories as they're written.

I could add a comment explaining what's going on here.